readme.md

```diff
@@ -25,13 +25,13 @@ hexRgb('#fff');
 //=> {red: 255, green: 255, blue: 255, alpha: 255}
 
 hexRgb('#4183c488');
-//=> {red: 65, green: 131, blue: 196, alpha: 136}
+//=> {red: 65, green: 131, blue: 196, alpha: 0.5333333333333333}
```

Comment:
Can you use a HEX alpha that doesn't end up with a lot of fractions? Better for presentational purposes.